packages/js-client-grpc/src/client-version.ts

```diff
@@ -1 +1,77 @@
 export const PACKAGE_VERSION = '1.12.0';
+
+interface Version {
+    major: number;
+    minor: number;
+}
+
+export const ClientVersion = {
+    /**
+     * Parses a version string into a structured Version object.
+     * @param version - The version string to parse (e.g., "1.2.3").
+     * @returns A Version object.
+     * @throws If the version format is invalid.
+     */
+    parseVersion(version: string): Version {
+        if (!version) {
+            throw new Error('Version is null');
+        }
+
+        let major = undefined;
+        let minor = undefined;
+        try {
+            [major, minor] = version.split('.', 2);
+            major = major.trim();
+            minor = minor.trim();
+            major = parseInt(major, 10);
+            minor = parseInt(minor, 10);
+        } catch (error) {
+            throw new Error(`Unable to parse version, expected format: x.y[.z], found: ${version}`);
```

Comment:
This try block shouldn't be necessary since `parseInt()` return NaN and doesn't throw.